gensim/test/test_drmm_tks.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
missing
- different training modes (that you have)
- "manual" training (when you don't pass all data in `__init__`)
- etc

Please have a look to w2v/fasttext/doc2vec tests for inspiration.